### PR TITLE
Feature/64 showing items on screen

### PR DESCRIPTION
**Pull request by PhoebeMitchell**
## Summary

Implements SHOW_ITEM command and adds HIDE_ITEM command. These allow a narrative script author to show or hide an image of an item on the screen. Adds ItemDisplay script to handle hiding and showing of item images.

## Before/after screenshots and/or animated gif

itemdisplaydemo

## Testing instructions

Open the "Inky-TestScene" scene and assign ShowItemDemo narrative script to the dialogue controller. Enter play mode and press X to cycle through the demo dialogue.

Test proof:
Screenshot 2021-09-19 at 16 17 26
Screenshot 2021-09-19 at 16 17 31

## Additional information

- `[x]` Changes UI
- `[x]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  
#74

### CODE REVIEW

File: Assets/Scripts/BGScenes/SceneController.cs [hunk 1]
```diff
@@ -80,9 +86,34 @@ public void ShakeScreen(float intensity)
...
+        Evidence evidence = _evidenceInventory.GetObjectFromAvailableObjects(item);
+        _itemDisplay.ShowItem(evidence.Icon, position);
+    }
+
+    /// <summary>
+    /// Hides the item currently being displayed on the screen.
+    /// </summary>
+    public void HideItem()
+    {
+        _itemDisplay.HideItem();
```
Comment (JuiceGrape) [inline Assets/Scripts/BGScenes/SceneController.cs:121]: Null check?

**Review by JuiceGrape — DISMISSED** [review]
Looks great

File: Assets/Scripts/BGScenes/SceneController.cs (hunk 1 above)
Comment (PhoebeMitchell) [reply Assets/Scripts/BGScenes/SceneController.cs:121]: Added

**Review by JuiceGrape — APPROVED** [review]

**Review by lyndsiWilliams — APPROVED** [review]
Looks good to me!

**Pull request merged**